Doc updates per conversation btw. Drew/Scott/Matt

matt-winkler opened the pull request:
https://fishtownanalytics.slack.com/archives/G01H940QRQ9/p1607988023003600

Also a question on how to references images in the markdown on our github.

## Description & motivation
<!---
Describe your changes, and why you're making them. Is this linked to an open
issue, a pull request on dbt core, etc?
-->

## To-do before merge
<!---
(Optional -- remove this section if not needed)
Include any notes about things that need to happen before this PR is merged, e.g.:
- [ ] Change the base branch
- [ ] Ensure PR #56 is merged
-->

## Pre-release docs
Is this change related to an unreleased version of dbt?
- [ ] Yes: please
    - update the base branch to `next`
    - add Changelog components: `<Changelog>[New/Changed] in v0.x.0</Changelog>`
    - add links to the "New and changed documentation" section of the latest [Migration Guide](../website/docs/docs/guides/migration-guide)
- [ ] No: please ensure the base branch is `current`
- [ ] Unsure: we'll let you know!

## Checklist
If you added new pages (delete if not applicable):
- [ ] The page has been added to `website/sidebars.js`
- [ ] The new page has a unique filename

If you removed existing pages (delete if not applicable):
- [ ] The page has been removed from `website/sidebars.js`
- [ ] An entry has been added to `_redirects`

matt-winkler committed:
Doc updates per conversation btw. Drew/Scott/Matt

https://fishtownanalytics.slack.com/archives/G01H940QRQ9/p1607988023003600

Also a question on how to references images in the markdown on our github.

drewbanin commented on website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-enterprise-sso-with-azure-active-directory.md:
```diff
@@ -78,11 +79,23 @@ the Redirect URI in the steps above, then skip ahead to step 8.
+11. Click **Add User/Group**
+12. Assign additional users and groups as-needed
+
+# Where do I go to upload the image as referenced in the path below? Not seeing this from navigating this git repo or searching for /img/ as a repository at the top level of the dbt org.
```
Images are served from the [website/static/](https://github.com/fishtown-analytics/docs.getdbt.com/tree/current/website/static) directory. So, the image above points to `/img/docs/dbt-cloud/dbt-cloud-enterprise/azure/azure-redirect-uri.png` which is [this file]((https://github.com/fishtown-analytics/docs.getdbt.com/blob/current/website/static/img/docs/dbt-cloud/dbt-cloud-enterprise/azure/azure-redirect-uri.png)

matt-winkler replied:
@drewbanin FYI fixed the image links

drewbanin replied:
@matt-winkler are those image changes in this PR? https://github.com/fishtown-analytics/docs.getdbt.com/pull/500

Would be good to merge these two branches together IMO, but LGTM once the images are being rendered in dev/staging :)

drewbanin reviewed (commented):
Thanks for making this PR! Let me know if you'd like a hand with regenerating images for these sections.... it's kind of a chore, but I think they're really helpful!

matt-winkler committed:
Update PR with links to images

Added the images to /static/, so cleaning up this PR to include the accurate image links.

matt-winkler committed:
Adding additional image links

Found some additional image links to clean up in the Troubleshooting Tips section

amychen1776 commented on website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-enterprise-sso-with-azure-active-directory.md:
```diff
@@ -78,11 +79,22 @@ the Redirect URI in the steps above, then skip ahead to step 8.
 
+### Adding Users to an Enterprise Application
+
+Once you've registered the application, the next step is to assign users to it. Add the users you want to be viewable to dbt with the following steps:
```
Could you link to our enterprise permissions? They need to create the dbt Cloud groups as well to map to those Azure AD groups.

matt-winkler replied:
@amychen1776 added a new header and paragraph for this immediately before this section.

amychen1776 commented on website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-enterprise-sso-with-azure-active-directory.md:
```diff
@@ -157,3 +169,11 @@ Users in your Azure AD account will now be able to log into the application
+
+Ensure that the domain name under which user accounts exist in Azure matches the domain supplied in the SSO configuration on the dbt side.
+
+<Lightbox collapsed="true" src="/img/docs/dbt-cloud/dbt-cloud-enterprise/azure/azure-get-domain.png" title="Obtaining the user domain from Azure" />
```
Did you want to include the Troubleshooting Tips in the same box as logging in?
![image](https://user-images.githubusercontent.com/46451573/103547733-cbf4fa00-4e72-11eb-8668-55bd299f9115.png)

amychen1776 commented on website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-enterprise-sso-with-azure-active-directory.md:
```diff
@@ -157,3 +169,11 @@ Users in your Azure AD account will now be able to log into the application
 by navigating to the URL:
 
 `https://cloud.getdbt.com/enterprise-login/<login-slug>`
```
If they're connecting to a single tenant or On-Prem instance, this URL would change the domain away from `cloud.getdbt.com`. Might be useful to flag this.

matt-winkler replied:
@amychen1776 Added a new commit with this called out.

amychen1776 replied:
Thank you!

matt-winkler committed:
Note domain change for single-tenant or on-prem

Per Amy's comment.

matt-winkler committed:
+ Enterprise Permissions link

Per Amy's comment.

matt-winkler committed:
Closes Enterprise Feature callout

Per Amy's comment.

matt-winkler commented:
Changes are now contained in https://github.com/fishtown-analytics/docs.getdbt.com/pull/502
